pom.xml

```diff
@@ -4,18 +4,18 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>no.nav</groupId>
     <artifactId>kafka-embedded-env</artifactId>
-    <version>2.5.0-SNAPSHOT</version>
+    <version>2.7.0-SNAPSHOT</version>
```

Comment:
Should be 2.7.0 to release?

Reply:
No, it's automatically set by the release workflow.